### PR TITLE
feat: add `rollup` spec

### PR DESCRIPTION
**Pull request by codedbyjordan**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
#1188 

**What is the new behavior (if this is a feature change)?**
Adds rollup spec

**Additional info:**

### CODE REVIEW

File: src/rollup.ts [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+  script: "ls",
+  postProcess: (output) => {
+    const files = output.split(/ +|\n/g);
+    const configFiles = files.filter((file) => file.endsWith(".config.js"));
+    return configFiles.map((configFile) => ({
+      name: configFile,
+      priority: 100,
+    }));
+  },
+};
```
Comment (mschrage) [inline src/rollup.ts:11]: We actually have some predefined helpers that you can use.

```
import { filepaths } from '@fig/autocomplete-generators'

...
generators: filepaths({ extension: [ "config.js" ]})
```

**Review by mschrage — CHANGES_REQUESTED** [review]
Looking good! Thank you for contributing - my one tiny suggestion is to use the helper functions rather than writing your own filepath generator.

File: src/rollup.ts [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+      args: {
+        name: "filename",
+        generators: configFiles,
+      },
+    },
+    {
+      name: ["-d", "--dir"],
+      args: {
+        name: "dirname",
+        description: "Directory for chunks (if absent, prints to stdout)",
```
Comment (fedeci) [inline src/rollup.ts:30]:
```suggestion
        description: "Directory for chunks (if absent, prints to stdout)",
        template: "folders",
```

**Review by fedeci — DISMISSED** [review]

**Comment by codedbyjordan** [comment]
@mschrage @fedeci committed all changes, didn't know about the helpers, tysm!

**Review by mschrage — APPROVED** [review]
Thank you so much for contributing @codedbyjordan! 🎉

**Comment by fedeci** [comment]
Great, thanks @codedbyjordan!

**Pull request merged**